src/schema/basic_classes.yaml

```diff
@@ -8,11 +8,22 @@ license: https://creativecommons.org/publicdomain/zero/1.0/
 prefixes:
   linkml: https://w3id.org/linkml/
   EFO: http://www.ebi.ac.uk/efo/
+  BFO: http://purl.obolibrary.org/obo/BFO_
+  rdfs: http://www.w3.org/2000/01/rdf-schema#
+  biolink: https://w3id.org/biolink/vocab/
+  owl: http://www.w3.org/2002/07/owl#
+
+default_curi_maps:
```

Comment:
I stopped using default curie maps under @cmungall 's guidance a long time ago. I'd like the three of us to discuss this before merging, but that would push it back to the first week of December.